code/controllers/stonedmc/subsystem/shuttle.dm

```diff
@@ -392,6 +393,9 @@ SUBSYSTEM_DEF(shuttle)
 	. = FALSE
 	// load shuttle template, centred at shuttle import landmark,
 	var/turf/landmark_turf = get_turf(locate(/obj/effect/landmark/shuttle_import) in GLOB.landmarks_list)
+	if(!landmark_turf)
+		to_chat(world, "no shuttle import landmark")

```

Comment:
Change this debug message not to be to the world at large?

Reply:
if theres no shuttle import landmark everything will break anyway

Reply:
That is fine, but that's what `CRASH` is for.

Reply:
no i mean EVERYTHING will break